docs/gettingstarted/development-version.md

```diff
@@ -0,0 +1,163 @@
+# Build development version of Kitodo 3.x
+
+## 1. System environment
+
+Download and install [Debian 9.4](https://cdimage.debian.org/debian-cd/9.4.0/amd64/iso-cd/).
+
+### Install sudo and reboot
+
+```
+su -c "apt install -y sudo && adduser $USER sudo && echo \"Defaults timestamp_timeout=300\" >> /etc/sudoers.d/timeout && reboot"
+```
+
+## 2. Build files for deployment
+
+### Add mysql.com 5.7 repository
+
+```
+sudo apt install -y dirmngr
+sudo apt-key adv --keyserver pgp.mit.edu --recv-keys 5072E1F5 && echo "deb http://repo.mysql.com/apt/debian/ stretch mysql-5.7" | sudo tee -a /etc/apt/sources.list.d/mysql-5.7.list
+```
+
+### Install packages openjdk-8, maven, mysql-community-server and zip
+
+```
+sudo debconf-set-selections <<< "mysql-community-server mysql-community-server/root-pass password "
+sudo debconf-set-selections <<< "mysql-community-server mysql-community-server/re-root-pass password "
+sudo apt update && sudo apt install -y openjdk-8-jdk maven mysql-community-server zip
+```
+
+### Build development version
+
+```
+wget https://github.com/kitodo/kitodo-production/archive/master.zip
+unzip master.zip && rm master.zip
+(cd kitodo-production-master/ && mvn clean package '-P!development')
+mv kitodo-production-master/Kitodo/target/kitodo-3*.war kitodo-3.war
+```
+
+Note: If you want to build a release version, you may want to set the version in pom.xml files before packaging
+
+### Remove duplicate version of bcprov dependency
+
+```
+zip -d kitodo-3.war "WEB-INF/lib/bcprov-jdk14-138.jar"
```

Comment:
Why not exclude it inside the pom?

Reply:
I remember that I got this Tomcat error message:
> Possible root causes include a too low setting for -Xss and illegal cyclic inheritance dependencies. The class hierarchy being processed was [org.bouncycastle.asn1.ASN1EncodableVector->org.bouncycastle.asn1.DEREncodableVector->org.bouncycastle.asn1.ASN1EncodableVector]

Excluding the dependency in the pom would be better but I will remove this step entirely because I can't reproduce the error right now.